modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Geolocation/Geolocation.es.js

```diff
@@ -55,6 +56,12 @@ const Geolocation = ({
 
 	return (
 		<div {...otherProps} className="ddm-geolocation field-labels-inline">
+			<div>
+				<ClayIcon symbol="geolocation" />
+
+				<span id={`address_label_${instanceId}`}></span>
+			</div>
+
 			{!disabled || viewMode ? (
 				<dl>
 					<dt className="text-capitalize"></dt>
```

Comment:
Hi @MirnaGama,

Can you move this `div` to be shown only if `!disabled || viewMode`? This way we will have the same behavior as 7.2.x version.

Something like:

```
{!disabled || viewMode ? (
	<>
		<div>
			<ClayIcon symbol="geolocation" />

			<span id={`address_label_${instanceId}`}></span>
		</div>

		<dl>
			<dt className="text-capitalize"></dt>

			<dd>
				<NoRender
					className="lfr-map"
					id={`map_${instanceId}`}
					style={{height: '280px'}}
				/>

				<input
					id={`input_value_${instanceId}`}
					name={name}
					type="hidden"
				/>
			</dd>
		</dl>
	</>
)
```

Thanks!